### DISCUSSION TITLE
How to forward the Authorisation Bearer to an API

### DISCUSSION THREAD

**Discussion by mielune** [Budibase/budibase#5951] [Help] [answered]
Hi BB community :)

I have a selft hosted BB instance with my own OIDC connector (plugin to my [gravitee.io](https://www.gravitee.io/) Access Management instance).
In my BB apps, I would like to cal some of my REST APIs secured by the same OIDC/Oauth2 gateway, so I would like to forward the Authrorisation Bearer get by BB to the REST API Call in my apps datasource. How can I do it ?

**Comment by melohagan** [comment, marked as answer]
Hi @mielune 

I don't think it's currently possible to pass along authorisation headers from the initial log-in. 
Here's a related discussion: https://github.com/Budibase/budibase/discussions/4218
